src/tcgwars/logic/impl/gen3/PopSeries2.groovy

```diff
@@ -379,6 +450,7 @@ public enum PopSeries2 implements LogicCardInfo {
           attackRequirement {}
           onAttack {
             damage 30
+            preventAllEffectsFromCustomPokemonNextTurn(thisMove, self, {it.EX})
```

Comment:
Does this prevent ex Pokémon only from applying effects to Celebi ex? It looks like it should be all Pokémon that are prevented from applying effects to it.

Reply:
Looks like the text got chopped off when it was generating the file.  I'll add the rest of the card text.  Here is an image of the card for reference. https://images-na.ssl-images-amazon.com/images/I/61mIljo8BCL._AC_.jpg